packages/testcontainers/src/container-runtime/clients/container/docker-container-client.ts

```diff
@@ -162,6 +162,7 @@ export class DockerContainerClient implements ContainerClient {
         follow: true,
         stdout: true,
         stderr: true,
+        tail: opts?.tail ?? -1,
```

Comment:
[Passing a negative number or a non-integer to --tail is invalid and the value is set to all in that case.](https://docs.docker.com/engine/reference/commandline/logs/#description)